src/app/shared/components/product/product-variation-display/product-variation-display.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="visible$ | async">
   <ng-container *ngIf="product$ | async as product">
     <div *ngFor="let attr of product.variableVariationAttributes" class="product-variation read-only">
-      <label>{{ attr.name }}:</label>
+      <span class="product-variation-info">{{ attr.name }}:</span>
```

Comment:
it would be nice, if the : could be created by scss and be removed from the template